client/packages/common/src/hooks/useUrlQuery/useUrlQuery.ts

```diff
@@ -4,24 +4,19 @@ interface UrlQueryObject {
   [key: string]: string | number | boolean;
 }
 
-// here you can override the parsers for specific query params
-// an example is the filter, which must allow filtering by numeric codes
-// if this is parsed as numeric, the query param changes filter=0300 to filter=300
-// which then does not match against codes, as the filter is usually a 'startsWith'
-const DefaultParsers: Record<
-  string,
-  (value: string) => string | boolean | number
-> = {
-  filter: (value: string) => value,
-};
-
-export const useUrlQuery = () => {
+interface useUrlQueryProps {
+  // an array of keys - the values of which should not be parsed before returning
+  // by default the value of parameters will be coerced to a number if !isNaN
+  // and to boolean if 'true' or 'false'. Specify keys here if you wish to opt out of this
+  skipParse?: string[];
+}
+export const useUrlQuery = ({ skipParse = [] }: useUrlQueryProps) => {
```

Comment:
```suggestion
export const useUrlQuery = ({ skipParse = [] }: useUrlQueryProps = {}) => {
```

Then you can get rid of those yucky empty object arguments you've used elsewhere ;)

Reply:
sure - though I don't mind the `({})` myself 🤷